e2e/jest/src/jest.test.ts

```diff
@@ -48,7 +48,7 @@ describe('Jest', () => {
     updateFile(
       `libs/${mylib}/setup.ts`,
       stripIndents`
-      const { registerTsProject } = require('nx/src/utils/register');
+      const { registerTsProject } = require('nx/src/plugins/js/utils/register');
```

Comment:
followup: This should use `@nrwl/js/src/internals`

Reply:
Something to discuss tomorrow...